index.js

```diff
@@ -696,19 +687,19 @@ PersonaClient.prototype.getProfileByGuid = function(guid, token, callback){
                 }
 
                 if(data){
-                  if (data.error) {
-                      callback(data.error, null);
-                  } else {
-                     callback(null, data);
-                  }
+                    if (data.error) {
+                        callback(data.error, null);
```

Comment:
opinionated: add return before `callback` (reduces the risk of a callback being called twice)